engine/executor/openapi.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"net/http"
+	"strings"
+	"sync"
+
+	libModel "github.com/pingcap/tiflow/engine/lib/model"
+)
+
+const jobAPIPrefix = "/api/v1/jobs/"
+
+func jobAPIBasePath(masterID libModel.MasterID) string {
```

Comment:
Use a `jobID` may be better. `MasterID` is the inner implement concept.

Reply:
Pls make an alias of `JobID = MasterID` in engine/model.

Reply:
OK.